Client/Frontend/Browser/Onboarding/Welcome/PrivacyEverywhereView.swift

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveUI
+import BraveShared
+
+struct PrivacyEverywhereView: View {
+    @Environment(\.colorScheme) private var colorScheme: ColorScheme
+    var dismiss: (() -> Void)?
+    var syncNow: (() -> Void)?
+
+    var body: some View {
+        VStack(spacing: 16) {
+            HStack {
+                Spacer()
+                Button {
+                    dismiss?()
+                } label: {
+                    Image(uiImage: #imageLiteral(resourceName: "privacy-everywhere-exit-icon"))
+                }
+                .frame(maxWidth: .infinity, alignment: .trailing)
+            }
```

Comment:
Think you misunderstood my previous comment, I meant that you can remove the HStack + Spacer entirely with `.frame(maxWidth: .infinity, alignment: .trailing)`.

I.e.

```suggestion
            Button {
                dismiss?()
            } label: {
                Image(uiImage: #imageLiteral(resourceName: "privacy-everywhere-exit-icon"))
            }
            .frame(maxWidth: .infinity, alignment: .trailing)
```

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4508/commits/a51ac89b29bda9b22be0eed68d06ded22e4dfad0